android/app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java

```diff
@@ -75,6 +79,15 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
+        boolean isLocal = SharedPreferencesUtil.getBoolean(getString(R.string.timezone_mode_key), false);
+        if (isLocal) {
+            String offset = new SimpleDateFormat("ZZZZ", Locale.getDefault()).format(System.currentTimeMillis());
```

Comment:
Move to DateUtils please

Reply:
Do you mean using the new Date and Time API ? I tried using them but they require API level 26 or above which is way too high.